### PR TITLE
bump dashboard version to 1.8.1 and add owner file

### PR DESCRIPTION
**Pull request by zouyee**
**What this PR does / why we need it**:
bump dashboard version ti 1.8.1 and add owner file

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```bump dashboard version ti 1.8.1 and add owner file

```

### CODE REVIEW

**Comment by roberthbailey** [comment]
@liggitt / @cjcullen for a sanity check on the updated rbac permissions

File: cluster/addons/dashboard/dashboard-rbac.yaml [hunk 1]
```diff
@@ -11,6 +11,10 @@ rules:
 - apiGroups: [""]
   resources: ["secrets"]
   verbs: ["create"]
+  # Allow Dashboard to create 'kubernetes-dashboard-settings' config map.
```
Comment (liggitt) [inline cluster/addons/dashboard/dashboard-rbac.yaml]: is there a reason not to let this be created by the addon manager... blanket create of configmaps in kube-system is really powerful. the same request was made for secrets when updating to 1.8.0 (https://github.com/kubernetes/kubernetes/pull/53046#discussion_r153808146), and the indication was that the dashboard would update to do that.

Comment (zouyee) [reply cluster/addons/dashboard/dashboard-rbac.yaml]: https://github.com/kubernetes/dashboard/pull/2667

**Comment by floreks** [comment]
If secret will be created empty (by addon manager) then in version 1.8.1 there is a bug that causes deadlock. https://github.com/kubernetes/dashboard/pull/2690

We will be doing a v1.8.2 release next week that already contains fix for that. I will update my PR #57326 then. This PR is actually a duplicate.

**Comment by zouyee** [comment]
close as duplicate

**Pull request closed without merging**